tests/integration/test_minidump.py

```diff
@@ -574,3 +574,62 @@ def test_crashpad_annotations(mini_sentry, relay_with_processing, attachments_co
         "annotations": ["dyld2 mode"],
         "type": "crashpad",
     }
+
+
+def test_chromium_stability_report(
```

Comment:
we should add two more tests
1. the minidump has an empty stability report
2. the minidump has a partially populated stability report

Maybe for 2 we can just add unit tests in `native.rs`.

Reply:
> 1. the minidump has an empty stability report

The other minidumps in the integration tests don't have the stability report stream. 

> 2. the minidump has a partially populated stability report

How do you mean?

We can rely on upstream `minidump` tests that we get either a valid `StabilityReport` or an error from the `minidump` crate. The conversion to the context types is now panic free because they're guarded against integer overflow.

The only thing we could potentially add a test for are the 4K multiplications?